index.html

```diff
@@ -1711,15 +1686,109 @@ <h2>Public Keys</h2>
     </section>
 
     <section>
-      <h2>Authentication</h2>
+      <h2>
+Authorization and Delegation
+      </h2>
+
+      <p>
+Authorization is the mechanism used to state how operations may be performed on
+behalf of the <a>DID subject</a>. Delegation is the mechanism that the subject
+uses to authorize others to act on their behalf.
+      </p>
+
+      <p class="note" title="Authorization vs authentication">
+Note that Authorization is separate from <a href="#authentication"></a>.
+This is particularly important for key recovery in the case of key loss,
+when the subject no longer has access to their keys, or key compromise,
+where the <a>DID controller</a>'s trusted third parties need to override
+malicious activity by an attacker. See Section
+<a href="#security-considerations"></a>.
+      </p>
+
+      <p>
+A DID document MAY include a <code>controller</code> property to indicate
+that there are <a>DID controller(s)</a> other than the <a>DID subject</a>.
+If so:
+      </p>
+
+      <dl>
+          <dt><dfn>controller</dfn></dt>
+          <dd>
+The value of the <code>controller</code> property MUST be a valid <a>DID</a>
+or an array of valid <a>DIDs</a>. The corresponding <a>DID document</a>(s)
+SHOULD contain authorization relations that explicitly permit the use of
```

Comment:
```suggestion
SHOULD contain <a>verification relationships</a> that explicitly permit the use of
```

Reply:
We may want to call "verification relationships" "authorization relationships" instead. It may be a better term.

Reply:
"authorization relationships" seems to more accurately describe the usage of these properties. However, I'm curious if using it may be more of a landmine than we want to encounter at this point because of the relationship between authorization and authentication (and how `authentication` is an "authorization relationship"). 

I'll open an issue to discuss this further and for this PR we can stick with verification relationships.